app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,96 @@
 <resources>
     <string name="app_name">HKSKH</string>
+    <string name="title_activity_shelter_scrolling">ShelterScrollingActivity</string>
+    <string name="large_text">
+        "Material is the metaphor.\n\n"
+
```

Comment:
What this be?

Reply:
I have no idea. I used a lot of the code from lab 32 and just made it work for our needs so its probably something Dr. Waters had in there